includes/GbWeiss.php

```diff
@@ -138,9 +140,43 @@ public function validateCredentials(): void
         }
     }
 
+    /**
+     *  Requests a new OAuthToken and stores the accessToken in
+     *  the ws_options table
+     *
+     *  @throws \Exception If the token could not be saved.
+     */
+    public function updateToken(): bool
```

Comment:
The boolean return is not necessary, since the function throws an exception if there is an error and never returns true. I would suggest to return void and keep throwing exceptions if someting went wrong.

Reply:
And I would like `updateAuthToken` better, since token by itself could be almost anyting.

Reply:
the updateTokenInDatabase returns true or false because the update_option method returns a bool as well